src/ray/gcs/tables.h

```diff
@@ -297,8 +297,7 @@ class TaskReconstructionLog : public Log<TaskID, TaskReconstructionData> {
   TaskReconstructionLog(const std::shared_ptr<RedisContext> &context,
                         AsyncGcsClient *client)
       : Log(context, client) {
-    pubsub_channel_ = TablePubsub_ACTOR;
```

Comment:
No need for `pubsub_channel_ = TablePubsub_TASK_RECONSTRUCTION;`?

Reply:
Yeah, because pubsub isn't enabled for this table. There's a comment about this for the `pubsub_channel_` declaration in the base class, but it's a bit hidden...I'll add a note to the class documentation.